package.json

```diff
@@ -73,7 +74,7 @@
         "moment": "^2.29.4",
         "nouislider": "11.0.3",
         "numeral": "^2.0.6",
-        "primevue": "4.0.0-beta.3",
+        "primevue": "^4.0.0-rc.2",
```

Comment:
Looks like this was an exact pin before, so I'd keep it that way for now, even though this whole pinning strategy is primed (ho ho ho) for a revamp in #11043.